.github/workflows/backstage-release.yaml

```diff
@@ -0,0 +1,72 @@
+name: Publish release on NPM
+
+on:
+  # When manually triggered
+  workflow_dispatch:
```

Comment:
Should this manual trigger have inputs ? Otherwise `actions/checkout@v4` will checkout the default branch `main`

Reply:
You can select the branch in GitHub UI and then trigger the workflow. No need for any inputs. ...unless there's something I am missing

Reply:
Should be good now